__tests__/server.test.js

```diff
@@ -89,4 +89,35 @@ describe("API endpoints", () => {
                 });
         });
     });
+    describe("GET: /api/articles", () => {
+        test("200 - sends an array of all the articles to the client", () => {
+            return request(app)
+                .get("/api/articles")
+                .expect(200)
+                .then((response) => {
+                    expect(response.body.articles.length).toBe(13);
+                    response.body.articles.forEach((article) => {
+                        expect(article).toHaveProperty("author");
+                        expect(article).toHaveProperty("title");
```

Comment:
You could use the second argument to `toHaveProperty` in combination with `expect.any(<type>)` here to have more confidence